### DISCUSSION TITLE
How to focus link and select move scroll on page?

### DISCUSSION THREAD

**Discussion by latipun7** [gdh1995/vimium-c#552] [Q&A] [answered]
I'd like to quickly jump to the link, but only to focus on it (not clicking), so I have a glimpse of the actual link before opening it. (Currently I can do that with going thru using tabs, but not ideal). Here's the screenshot of what I meant:
![link](https://i.imgur.com/nscLTZ3.png)

Then, my second question is, how to move between panel that has different scroll?
![scroll](https://i.imgur.com/OheDnCi.png)

Thank you.

**Comment by gdh1995** [comment, marked as answer]
You may use `LinkHints.activateHover` or `LinkHints.activateFocus` to "select" an element without clicking it.

Currently there's no browser API for extensions to show the bottom tip. While you may:
1. press tab to switch focus into near elements (or even switch back), to trigger the tip
2. in v1.97.0 of Vimium C, the two commands will support a boolean parameter named `showUrl`, and Vimium C will show URL in its own HUD

As for moving among scrollable elements, you may select an outer box element using `LinkHints` mode, and then `scroll*` will work on it.
- If an element is not clickable but scrollable, its mark text will be dark red.
- If a mark is covered by another one, you may press `<space>` to change the order and see the below one
  - this require the option `Use the link's name and characters for link-hint filtering` is disabled.

If you can not find a hint mark for the whole scrollable box, then you may use `LinkHints.activateHover` to select any one of its descendent elements, and `scroll*` commands will also auto recognize it.

**Reply by latipun7** [reply]
Thank you. Turned out I just lack of reading the wiki 🙇‍♂️

For now, all my questions solved by following your directions.
